tests/libs/datasets/timeseries_test.py

```diff
@@ -597,15 +598,18 @@ def test_write_read_wide_dates_csv_compare_literal(tmpdir):
 
     # Check that a file without the demographic_bucket column (as written before
     # https://github.com/covid-projections/covid-data-model/pull/1021) can be read.
-    pointer.path_wide_dates().write_text(
-        "                  location_id,variable,provenance,2020-04-03,2020-04-02,2020-04-01\n"
-        "           iso1:us#iso2:us-as,   cases,          ,       300,       200,       100\n"
-        "           iso1:us#iso2:us-as,icu_beds,   pt_src1,         4,         2,         0\n"
-        "iso1:us#iso2:us-ca#fips:06075,   cases,          ,       310,       210,\n"
-        "iso1:us#iso2:us-ca#fips:06075,  deaths,   pt_src2,          ,         2,         1\n".replace(
-            " ", ""
+    data = pd.read_csv(
+        io.StringIO(
+            "                  location_id,variable,provenance,2020-04-03,2020-04-02,2020-04-01\n"
+            "           iso1:us#iso2:us-as,   cases,          ,       300,       200,       100\n"
+            "           iso1:us#iso2:us-as,icu_beds,   pt_src1,         4,         2,         0\n"
+            "iso1:us#iso2:us-ca#fips:06075,   cases,          ,       310,       210,\n"
+            "iso1:us#iso2:us-ca#fips:06075,  deaths,   pt_src2,          ,         2,         1\n".replace(
+                " ", ""
+            )
         )
     )
+    data.to_csv(pointer.path_wide_dates(), index=False, compression="gzip")
```

Comment:
Same as above comment, using pandas `to_csv` to take advantage of the `compression="gzip"` argument